packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -205,9 +206,23 @@ export class OdspDocumentService implements IDocumentService {
             const websocketTokenPromise = requestWebsocketTokenFromJoinSession
                 ? Promise.resolve(null)
                 : this.getWebsocketToken!(options);
+            const joinSessionPromise = this.joinSession(requestWebsocketTokenFromJoinSession).catch((e) => {
+                const code = e?.response ? JSON.parse(e?.response)?.error?.code : undefined;
```

Comment:
@wes-carlson , can you please help me understand where e.response is coming from? 
I fail to see who sets this property (even though I do see it's being set, when errors happen in stress tests).

Reply:
`fetchHelper()` in odspUtils.ts passes the entire response to `throwOdspNetworkError()`, which in turn passes it to `createOdspNetworkError()`.

https://github.com/microsoft/FluidFramework/blob/4aeb50159e94175bb5c63a30eedee8ba04796471/packages/drivers/odsp-driver/src/odspUtils.ts#L110

https://github.com/microsoft/FluidFramework/blob/4aeb50159e94175bb5c63a30eedee8ba04796471/packages/utils/odsp-doclib-utils/src/odspErrorUtils.ts#L136

Reply:
Thanks, I missed that it was responseText, I was tracing responce object and could not figure out how it gets there :)

Reply:
Yet another case study backing up #6485 :)